array_experimental.go

```diff
@@ -94,3 +98,26 @@ func (cp *ConsolidationPlan) DumpJSON() (string, error) {
 
 	return json, nil
 }
+
+// ConsolidateFragments consolidates an explicit list of fragments in an array into a single fragment.
+// You must first finalize all queries to the array before consolidation can
+// begin (as consolidation temporarily acquires an exclusive lock on the array).
+func (a *Array) ConsolidateFragments(config *Config, fragmentList []string) error {
+	if config == nil {
+		return fmt.Errorf("Config must not be nil for Consolidate")
+	}
+
+	curi := C.CString(a.uri)
+	defer C.free(unsafe.Pointer(curi))
+
+	list, freeMemory := cStringArray(fragmentList)
+	defer freeMemory()
+
+	ret := C.tiledb_array_consolidate_fragments(a.context.tiledbContext, curi, (**C.char)(unsafe.Pointer(&list[0])), C.uint64_t(len(list)), config.tiledbConfig)
```

Comment:
`(unsafe.Pointer(&list[0]))` can be `slicePtr(list)`.

Reply:
Done